package.json

```diff
@@ -12,9 +12,6 @@
     "semver": "^5.3.0",
     "jszip": "2.5.x"
   },
-  "bundleDependencies": [
-    "tar"
```

Comment:
tar is needed to run the install script. npm doesn't guarantee that a packages deps are installed before its install script is run, so the only safe way for an install script to use any deps is to have them be bundled. Whether any particular install sees this or not depends on race conditions, so while not bundling the install script's deps may work most of the time, for most people, it fails for us in CI, where we do lots of runs, across lots of systems.